makefile

```diff
@@ -13,6 +13,7 @@ endif
 endif
 endif
 
+
```

Comment:
Can you add a `bump` target:

```
bump:
ifneq ($(shell git diff --staged),)
	@git diff --staged
	@$(error You have uncommitted changes. Push or discard them to continue)
endif
	@./scripts/bump-version.sh $(VERSION)
```

Reply:
This allows you to do `make VERSION=1.2.3 bump` like other repos.

Also, can you update the CONTRIBUTING.md guide?